packages/addresses/src/address.ts

```diff
@@ -45,12 +45,16 @@ export function isAddress(putativeAddress: string): putativeAddress is Address<t
     }
     // Slow-path; actually attempt to decode the input string.
     const base58Encoder = getMemoizedBase58Encoder();
-    const bytes = base58Encoder.encode(putativeAddress);
-    const numBytes = bytes.byteLength;
-    if (numBytes !== 32) {
-        return false;
+    try {
+        const bytes = base58Encoder.encode(putativeAddress);
+        const numBytes = bytes.byteLength;
+        if (numBytes !== 32) {
+            return false;
+        }
+        return true;
+    } catch (e) {
+        return false
```

Comment:
Heh, we already had this pretty verbose, but what do you think about simplifying?

```suggestion
    try {
        return base58Encoder.encode(putativeAddress).byteLength === 32;
    } catch {
        return false;
    }
```

Reply:
Sure looks fine helps to simplify.